gnomad/utils/sparse_mt.py

```diff
@@ -280,6 +280,7 @@ def get_as_info_expr(
     array_sum_agg_fields: Union[
         List[str], Dict[str, hl.expr.ArrayNumericExpression]
     ] = INFO_ARRAY_SUM_AGG_FIELDS,
+    alt_alleles_range_array: str = "alt_alleles_range_array",
```

Comment:
For consistency with other arguments that take field names, maybe name this `alt_alleles_range_array_field`?